### ISSUE TITLE
Missing Privacy policy / terms of use / accessibility

### ISSUE THREAD

**Issue by mtanneau** [gt-sse-center/project-showcase#67]
The [Privacy Policy](https://gt-sse-center.github.io/project-showcase/#), [Terms of Use](https://gt-sse-center.github.io/project-showcase/#) [Accessibility](https://gt-sse-center.github.io/project-showcase/#) at the far bottom of the page all point to the main page itself.

<img width="1540" height="76" alt="Image" src="https://github.com/user-attachments/assets/b5dc430b-aef2-49ba-bf5d-f15523b44154" />
